en/12_Project_Governance/05_Release_policy.md

```diff
@@ -0,0 +1,87 @@
+
+> [!NOTE]
+> Refer to our [definition of public API](/project_governance/public_api/).
+
+The purpose of this policy is to provide clear guidance on the types of releases, their cadence, scope, and support lifecycle, ensuring predictability and stability for our users and developers.
+
+## Release types and cadence
+
+Silverstripe CMS adheres to [Semantic versioning](https://semver.org/) for its release philosophy.
+
+### Major releases `(x.0.0)`
+
+Major releases include architectural updates, allow breaking changes and major dependency changes.
+
+- **Purpose:** To deliver breaking changes and architectural updates that cannot be shipped in minor releases due to backward compatibility constraints.
+
+- **Backward compatibility:** Major releases are **not backward compatible**.
+
+- **Frequency:** New Silverstripe CMS major versions are released every **two years**.
+
+- **Tagging period:** New major releases are typically tagged as stable between **April and June** of their release year.
+
+- **Scope:** This policy applies to all Silverstripe CMS commercially supported modules. At launch, the list of commercially supported modules for that specific major release line will be explicitly defined.
+
+- **Pre-release period:** A stable major release is always preceded by a **beta period of at least three months** to allow for extensive testing and feedback.
+
+- **Dependency alignment:** At launch, Silverstripe CMS major releases aim to:
+
+  - Support all PHP versions currently in full support.
+  - Have all PHP and JavaScript dependencies on their latest stable versions.
+
+- **Upgrade path:** Silverstripe CMS commits to providing a clear upgrade path to new major releases, including comprehensive documentation on all breaking changes and migration guides.
+
+### Minor releases `(x.y.0)`
+
+Minor releases provide ongoing value and improvements within a stable major release line. They are backwards compatible within the major release line and designed to be easy to upgrade to.
+
+- **Purpose:** To ship new features, enhancements, performance improvements, and bug fixes that cannot be shipped in a patch releae due to API additions.
+
+- **Backward compatibility:** Minor releases are **backward compatible** within their major release line. They will not introduce breaking changes.
+
+- **Frequency:** Minor releases are shipped every **six months** within an active major release line, typically targeted for **April and October**. There will be **five** minor releases per major release.
+
+- **Pre-release period:** Minor releases are typically preceded by a **beta period of approximately six weeks** and a **release candidate (RC) period of approximately two weeks**. A feature freeze is implemented after the beta phase.
+
+### Patch releases `(x.y.z)`
+
+Patch releases address critical issues swiftly to maintain stability and security.
+
+- **Purpose:** To deliver bug fixes and security updates.
```

Comment:
```suggestion
- **Purpose:** To deliver bug fixes
```
Security patches should be treated in their own section since those do have specific release windows and have a different scope.